cmd/pod-scaler/admission_test.go

```diff
@@ -28,6 +29,17 @@ import (
 	"github.com/openshift/ci-tools/pkg/testhelper"
 )
 
+type mockReporter struct {
+	client *http.Client
+	state  string
```

Comment:
rather than a string, I would just use a `bool` here. you can just set it to `true` when the method is called.